src/pipecat/services/aws.py

```diff
@@ -33,37 +33,79 @@
 
 
 def language_to_aws_language(language: Language) -> str | None:
-    language_map = {
+    AWS_LANGUAGE_MAP = {
```

Comment:
for consistency, should this be changed back? or should all the others change to match?

Reply:
Thanks for catching this. I'm going to switch back to `language_map`.